test/test_getctydata.c

```diff
@@ -60,6 +61,59 @@ char *full_prefix(char * call) {
     return pfx->pfx;
 }
 
+void test_xspeed(void **state) {
```

Comment:
this could be removed (added for reference), or at least N reduced, as it takes precious seconds on each Travis run.
(not sure what is the current state at Travis, they were supposed to stop the free service end of 2020)

Reply:
> this could be removed (added for reference), or at least N reduced, as it takes precious seconds on each Travis run.

I will look to keep it around but have a compile time switch (a simple #define) to make it a no-op if not needed.

> (not sure what is the current state at Travis, they were supposed to stop the free service end of 2020)

Just copied the following from travis-ci.com: 

"Testing your open source projects is always 100% free!"